src/box/alter.cc

```diff
@@ -2005,11 +2010,13 @@ on_replace_dd_space(struct trigger * /* trigger */, void *event)
 		space_cache_replace(NULL, space);
 		/*
 		 * Do not forget to update schema_version right after
-		 * inserting the space to the space_cache, since no
+		 * inserting the space to the space_cache
+		 * (unless space is meta-temporary), since no
 		 * AlterSpaceOps are registered in case of space
 		 * create.
 		 */
-		box_schema_version_bump();
+		if (!space_is_meta_temporary(space))
+			box_schema_version_bump();
```

Comment:
I don't think you can skip the version bump. Connectors won't see those spaces if they are created after connection is established.

Reply:
Reverted back to updating the schema version